drop-transfer/examples/udrop.rs

```diff
@@ -58,7 +58,7 @@ async fn listen(
     let mut storage = drop_transfer::StorageDispatch::new(&storage);
 
     while let Some(ev) = rx.recv().await {
-        if let Err(e) = storage.handle_event(&ev).await {
+        if let Err(e) = storage.handle_event(&ev) {
```

Comment:
TBH we might want to use the `tokio::task::block_in_place()` here

Reply:
done